sync/sync-lib/build.gradle

```diff
@@ -0,0 +1,38 @@
+plugins {
+    id 'com.android.library'
+    id 'kotlin-android'
+    id 'com.squareup.anvil'
+}
+
+apply from: "$rootProject.projectDir/gradle/android-library.gradle"
+
+android {
+
+    defaultConfig {
+        defaultConfig {
+            ndk {
+                abiFilters 'arm64-v8a', 'x86', 'x86_64'
```

Comment:
On architecture is missing here. Something to fix before merging, but not a blocker for the review.